### PR TITLE
Fix issue 131 - support python 3.9

### PR DESCRIPTION
**Pull request by JanBargeman**

### CODE REVIEW

File: .github/workflows/unit_tests.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
           - build: windows
             os: windows-latest
             SKIP_LIGHTGBM: False
-        python-version: [3.6, 3.7, 3.8]
```
Comment (Matgrb) [inline .github/workflows/unit_tests.yml:27]: Lets also include it into the workflow in cronjob_unit_tests.yml file

**Review by Matgrb — CHANGES_REQUESTED** [review]
Looks good, only one minor change and we can merge.

File: .github/workflows/unit_tests.yml (hunk 1 above)
Comment (JanBargeman) [reply .github/workflows/unit_tests.yml:27]: I've implemented your suggestion and pushed it to here!

**Review by Matgrb — APPROVED** [review]
Great, thanks!

**Pull request merged**